app/script/view_model/ConversationInputEmojiViewModel.js

```diff
@@ -49,10 +49,29 @@ z.ViewModel.ConversationInputEmojiViewModel = class ConversationInputEmojiViewMo
       $(event.currentTarget).addClass('selected');
     });
 
-    fetch('/image/emoji.tsv')
-      .then((response) => response.text())
-      .then((text) => {
-        this.emoji_dict = text.split('\n').filter((event) => event.length > 0);
+    fetch('/image/emoji.json')
+      .then((response) => response.json())
+      .then((json) => {
+        this.emoji_dict = Object.entries(json)
```

Comment:
Good catch @gregor. In fact, I think it was a wrong choice to use dictionary for storing emojis, as we always consume them as arrays. So I changed the code, should be simpler now and not use any experimental features 🙂